client/styles.css

```diff
@@ -1,340 +0,0 @@
-}
-
-#search-icon{
-    display: flex;
-    align-items: center;
-    margin-left: 15px;
-}
-
-#mic-icon{
-    display: flex;
-    align-items: center;
-    margin-right: 23px;
-}
-
-#mic-icon:hover{
-    cursor: pointer;
-}
-
-#search-icon img{
-    opacity: 0.5;
-}
-
-/* Styling Search Buttons */
-.buttons-container{
-    display: flex;
-    justify-content: center;
-    margin-top: 20px;
-}
-
-.button{
-    background-color: #f8f9fa;
-    border: 1px solid #f8f9fa;
-    border-radius: 4px;
-    color: #3c4043;
-    font-family: arial, sans-serif;
-    font-size: 14px;
-    margin: 11px 4px;
-    padding: 0 8px;
-    line-height: 27px;
-    height: 36px;
-    min-width: 54px;
-    text-align: center;
-    cursor: pointer;
-    user-select: none;
-}
-
-#google-search:hover{
-    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
-}
-
-#feeling-lucky{
-    width: 130px;
-}
-
-#feeling-lucky:hover{
-    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
-}
-
-/* Styling Link Section */
-#link-section{
-    display: flex;
-    flex-direction: column;
-    width: 30%;
-    justify-content: center;
-    margin: 2% auto;
-
-}
-
-.link-container{
-    width: 600px;
-    height: 120px
-}
-
-.link-container a{
-    color:#202124;
-    text-decoration: none;
-    padding-bottom: 10px;
-}
-
-.link-container h3{
-    margin-top: 6px;
-    width: 13%;
-    margin-bottom: 6px;
-    color: #1a0dab;
-    white-space: nowrap;
-}
-
-.link-container h3:hover{
-    text-decoration: underline;
-    text-decoration-thickness: 2px;
-    cursor: pointer;
-}
-
-.link-container div{
-    color: #4d5156;
-}
-
-/* Footer Styling */
-.footer{
-    position: fixed;
-    bottom: -30px;
-    height: 96px;
-    width: 100%;
-}
-
-.footer-banner{
-    display: flex;
-    position: fixed;
-    height: 48px;
-    width: 100%;
-    background: #f2f2f2;
-    align-items: center;
-    padding: 15px 30px;
-    color: rgba(0,0,0,.54);
-    border-bottom: 1px solid #dadce0;
-    margin-top: -30px;
-    font-size: 15px;
-    font-family: arial, sans-serif;
-}
-
-.footer-carbon{
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    margin-top: 18px;
-    height: 48px;
-    background-color: #f2f2f2;
-    padding: 15px 20px;
-    color: #70757A;
-}
-
-.footer-carbon a{
-    color: #70757A;
-    text-decoration: none;
-    position: absolute;
-    left: 0;
-    right: 0;
-    margin: 0 auto;
-    width: 182px;
-    /* margin-right: 10%; */
-}
-
-#leaf{
-    margin-right: 5px;
-}
-
-#footer-ul-1{
-    display: flex;
-    list-style: none;
-}
-
-#footer-ul-1 li{
-    padding: 15px;
-}
-
-#footer-ul-1 li:hover{
-    cursor: pointer;
-    text-decoration: underline;
-}
-
-#footer-ul-2{
-    display: flex;
-    list-style: none;
-}
-
-#footer-ul-2 li{
-    padding: 15px;
-}
-
-#footer-ul-2 li:hover{
-    cursor: pointer;
-    text-decoration: underline;
-}
```

Comment:
really nice job on the styling - would like to see some more responsive elements, e.g., results page is kind of broken upon resizing
maybe define some variables for the reused colours